views/client.go

```diff
@@ -19,6 +19,8 @@ type Metric struct {
 
 	// breakdown fields
 
+	CreatorId   data.Nullable[string] `json:"creatorId,omitempty"`
+	ViewerId    data.Nullable[string] `json:"viewerId,omitempty"`
```

Comment:
The convention in go is to keep abbreviations in all upper-case. Would you mind making these `CreatorID` and `ViewerID` instead?
```suggestion
	CreatorID   data.Nullable[string] `json:"creatorId,omitempty"`
	ViewerID    data.Nullable[string] `json:"viewerId,omitempty"`
```